tests/scripts/analyze_outcomes.py

```diff
@@ -690,9 +690,9 @@ def main():
             test_args = KNOWN_TASKS[task]['args']
             main_results = test_function(main_results, options.outcomes, test_args)
 
-        main_results.info("Overall results:\n" + \
-                      "{} warnings\n".format(main_results.warning_count) + \
-                      "{} errors\n".format(main_results.error_count))
+        main_results.info("Overall results: " + \
```

Comment:
Style: I don't think we need the `\` for line continuation inside a `(`.

Reply:
Also style: why not `main_results.info("Overall results: {} warnings and {} errors.", main_results.warning_count, main_results.error_count)`? (Taking advantage of `info()`'s signature.)